src/pipecat/services/google/google.py

```diff
@@ -720,6 +782,14 @@ async def _process_context(self, context: OpenAILLMContext):
         except Exception as e:
             logger.exception(f"{self} exception: {e}")
         finally:
+            if grounding_metadata is not None and isinstance(grounding_metadata, dict):
+                llm_search_frame = LLMSearchResponseFrame(
```

Comment:
One nice improvement would be to create an RTVI message so we can share this response with the RTVI clients. What do you think? I can create a follow-up PR to implement the message.

Reply:
Maybe something like: `RTVIBotLLMSearchResponse`. 

What do you think ?

Reply:
I have included it in this PR: https://github.com/pipecat-ai/pipecat/pull/1051